webrtc/RTCDTMFSender-helper.js

```diff
@@ -44,14 +47,11 @@ function createDtmfSender(pc = new RTCPeerConnection()) {
             return;
           }
           ++counter;
-          step_timeout(checkfunc, 10);
+          step_timeout(checkCanInsertDTMF, 10);
         }
-      };
-      checkfunc();
+      }, 0);
     });
   }).then(() => {
-    assert_true(dtmfSender.canInsertDTMF,
```

Comment:
this assertion used to be always true, the promise would have been rejected otherwise.